src/content/en/fundamentals/performance/navigation-and-resource-timing/index.md

```diff
@@ -0,0 +1,689 @@
+
+// Time to First Byte (TTFB)
+var ttfb = pageNav.responseStart - pageNav.requestStart;
+```
+
+Now that we've covered the most vital performance metrics provided by Navigation
+and Resource Timing, let's take a brief look at some of their less consequential
+metrics.
+
+### The other stuff
+
+There's a ton of metrics and properties in these APIs, but not _all_ of them are
+crucial. That said, you should at least be aware of them.
+
+#### Document unloading
+
+Unloading is when the browser performs cleanup prior to loading a new page. This
+isn't a big deal in typical circumstances, but it could be worth quantifying if
+you have code (especially from third parties) running in a [`unload` event
+handler](https://developer.mozilla.org/en-US/docs/Web/Events/unload) that holds
+up the next page from rendering. If that describes your situation, you'll want
+to pay attention to the
+[`unloadEventStart`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/unloadEventStart)
+and
+[`unloadEventEnd`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/unloadEventEnd)
+metrics.
+
+_Note: Unload metrics are exclusive to Navigation Timing._
+
+#### Redirects
+
+OK, page redirects aren't _totally_ inconsequential, but they might not be
+something you run into very often. Still, redirects add latency to requests, so
+measuring them may be worth the effort.
+
+<figure>
+  <img src="images/figure-3-1x.png" srcset="images/figure-3-2x.png 2x,
+images/figure-3-1x.png 1x" alt="A redirect chain visualized in Chrome's
+developer tools.">
+  <figcaption><b>Figure 3</b>. A redirect chain shown in Chrome’s developer
+tools.</figcaption>
+</figure>
+
+If you're concerned about redirects, both APIs expose the
+[`redirectStart`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/redirectStart)
+and
+[`redirectEnd`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/redirectEnd)
+metrics, which makes measuring redirect time a trivial task.
+
+#### Document processing
+
+When HTML documents load, the browser takes time to process them into the DOM.
+Navigation Timing exposes metrics to measure this process, but they aren't
+usually consequential unless _perhaps_ you're serving huge documents. If you're
+interested in document processing metrics, they're
+[`domInteractive`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/domInteractive),
+[`domContentLoadedEventStart`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/domContentLoadedEventStart),
+[`domContentLoadedEventEnd`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/domContentLoadedEventEnd),
+and
+[`domComplete`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming/domComplete).
+
+_Note: Document processing metrics are exclusive to Navigation Timing._
```

Comment:
`Note:`'s are [special callouts](https://developers.google.com/web/resources/markdown-syntax#callouts), don't wrap in italics.